Source/JavaScriptCore/b3/air/AirGenerate.cpp

```diff
@@ -109,47 +108,27 @@ void prepareForGeneration(Code& code)
     // https://bugs.webkit.org/show_bug.cgi?id=150456
     
     eliminateDeadCode(code);
-
-    size_t numTmps = code.numTmps(Bank::GP) + code.numTmps(Bank::FP);
-    if (code.optLevel() == 1 || numTmps > Options::maximumTmpsForGraphColoring()) {
```

Comment:
Doesn't this change regression the case `numTmps > Options::maximumTmpsForGraphColoring()`?